pkg/metrics/pool.go

```diff
@@ -0,0 +1,60 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	poolGetCounterMetricsName = "pool_get_total"
```

Comment:
BTW, I found that metrics name is not specific. Users cannnot get it directly.
1. what is pool used to do?
2. get or put what?

Reply:
It records the times of `Get` and `Put` method called. Do you have better suggestion?

Reply:
How about adding a `lua_vm` prefix?

Reply:
This pool can be used not only in lua vm. And `luavm` is added in the pool name.
```
pool_get_total{from="pool",name="luavm",type="fixedpool"} 1
```

Reply:
I think it's hard to use one metrics covering multiple scenes. For example, the labels that need to be used will not be exactly the same.

Reply:
> I think it's hard to use one metrics covering multiple scenes. For example, the labels that need to be used will not be exactly the same.

This metric is recording the operation of pool, not the luavm. This metric covers all the operations of pool, I think it's enough.

Reply:
How about `pool_get_operation_total`?

Reply:
@Poor12 How do you think?